docs/changelog/0024-define-management-commands.md

```diff
@@ -1,48 +0,0 @@
-<!--THIS FILE IS AUTOGENERATED: DO NOT EDIT-->
```

Comment:
The correct version of this file already exists (0024-define-management-commands.md)